pkg/controller/pxc/controller.go

```diff
@@ -914,6 +913,19 @@ func (r *ReconcilePerconaXtraDBCluster) reconcilePDB(spec *api.PodDisruptionBudg
 // ErrWaitingForDeletingPods indicating that the stateful set have more than a one pods left
 var ErrWaitingForDeletingPods = fmt.Errorf("waiting for pods to be deleted")
```

Comment:
Error is not used anywhere except one func. So no need to create var.
I konw it's not related ti this PR, just saw it here

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/704/commits/8a0bb3f16ef8c17b9c42c81e791473e977fc074d